notebook.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>This is great, look at you remembering how to format strings in Python! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meg-delaunay/reviewnb-notebook-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>